<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::DragOnly</enum>
   </property>
   <item>
    <property name="text">
     <string>Понедельник</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Вторник</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Среда</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Четверг</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Пятница</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Суббота</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Воскресенье</string>
    </property>
   </item>
  </widget>
  <widget class="QListWidget" name="listWidget_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::DropOnly</enum>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::TargetMoveAction</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::DropOnly</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>add</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
